src/tagging/components/tagging.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'patternfly-react';
+import TagModifier from './tagModifier';
+import TagView from './tagView';
+import CategoryModifier from './categoryModifier';
+import ValueModifier from './valueModifier';
+
+class Tagging extends React.Component {
+  onTagValueChange = (selectedTagValue) => {
+    const action = { tagCategory: this.props.selectedTagCategory, tagValue: selectedTagValue };
+    if (this.findSelectedCat(this.props.selectedTagCategory).singleValue) {
```

Comment:
When limited to single select, we are replacing the value, correct?  Just want to make sure we aren't forcing the user to delete previous value before adding a new one

Reply:
Yes, it just replace the value. User doesn't have to delete previous value.